kornia/utils/image.py

```diff
@@ -154,22 +158,43 @@ def forward(self, x: np.ndarray) -> torch.Tensor:
         return image_to_tensor(x, keepdim=self.keepdim)
 
 
-def perform_keep_shape(f):
+def perform_keep_shape_image(f):
```

Comment:
```suggestion
def perform_keep_shape_image(f: Callable) -> torch.Tensor:
```
add typing wherever makes sense to improve code readability